app/models/user.rb

```diff
@@ -1,5 +1,7 @@
+
 class User < ApplicationRecord
-  
+  attr_accessor :activation_token
+  #before_save :downcase_email   <<<--- is this needed?
```

Comment:
Yes, this is needed. We've had issues before with accounts that did not have downcased emails. Feel free to implement it.